pkg/apis/flowcontrol/types.go

```diff
@@ -144,6 +142,27 @@ type PolicyRuleWithSubjects struct {
 	Rule PolicyRule
 }
 
+// PolicyRule holds information that describes a policy rule, but does not contain information
+// about who the rule applies to or which namespace the rule applies to.
```

Comment:
I see this type definition was moved to a better place and that is good enough.  Reading it fresh I notice some problems in some of the field comments; they can be fixed in follow-up.